shell/platform/darwin/macos/framework/Source/FlutterThreadSynchronizer.mm

```diff
@@ -3,6 +3,7 @@
 
 #import <QuartzCore/QuartzCore.h>
 #include <mutex>
```

Comment:
While you're in here, Add a blank line between the C header and the C++ headers :)